{{ cookiecutter.repo_name }}/docs/user_guide/loading_environment_variables.md

```diff
@@ -0,0 +1,77 @@
+# Loading environment variables
+
+We use [`direnv`][direnv] to load environment variables, as it ensures you only have project-specific variables loaded
+_when you are inside the project_, otherwise these variables are not loaded. This can prevents accidental conflicts
+with identically named variables.
+
+```{contents}
+:local:
+:depth: 2
+```
+
+## Using `direnv`
+
+To load the environment variables, first [install `direnv`](#installing-direnv){% if cookiecutter.create_secrets_file == "Yes" %}, and make sure you have a `.secrets` file to [store secrets and credentials](#storing-secrets-and-credentials){% endif %}; then:
```

Comment:
I assume all references to a secrete file option will have been edited in the separate branch / future PR so I am not commenting on any here or elsewhere :)